test/cql-pytest/test_uda.py

```diff
@@ -230,3 +230,37 @@ def test_drop_keyspace_with_uda(scylla_only, cql):
     assert aggregates_before.result - aggregates_after.result == 1
     with pytest.raises(ConfigurationException, match="Cannot drop non existing keyspace"):
         cql.execute(f"DROP KEYSPACE {ks}")
+
+# Test that replacing the state function, reduce function or the final function succesfully changes the function used by the aggregate
+def test_replace_sfunc_ffunc(cql, test_keyspace, cassandra_bug):
```

Comment:
Please add a short comment to explain why this is a cassandra_bug (that Cassandra uses the old UDF even after it's replaced).

Reply:
Ok